backend/follows/api/views.py

```diff
@@ -14,11 +14,13 @@
 from follows.models import Follow
 from profiles.models import Profile
 from users.models import User
-from utils.jwt_payload import getJWTPayload
 
 class createFollowAPI(APIView):
-    def post(self, request):
-        serializer = FollowSerializer(data=request.data)
+    def post(self, request, pk):
+        follower = Profile.objects.get(user_id=request.user.id)
+        following = Profile.objects.get(id=pk)
+
+        serializer = FollowSerializer(data={'follower_id':follower.id, 'following_id':following.id})
```

Comment:
```suggestion
    def post(self, request, pk):
        following = Profile.objects.get(id=pk)

        serializer = FollowSerializer(data={'follower_id':request.user.id, 'following_id':following.id})
```
no need to verify follower since request is already legit with the use of jwt auth